build-env/docker/docker-debian-11-bullseye/apt.conf.d/10-no-check-valid-until

```diff
@@ -0,0 +1,2 @@
+# ignore "valid until" timestamp so that installing from snapshot will work
+Acquire::Check-Valid-Until "0";
```

Comment:
We don't need this since we're not using snapshot for bullseye like we did for buster.  fixup PR against debian-11 branch: https://github.com/PelionIoT/distro-pelion-edge/pull/214